drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -253,11 +434,11 @@ local function device_init(driver, device)
       -- create child devices as needed for multi-switch devices
       initialize_switch(driver, device)
     end
-    device:set_component_to_endpoint_fn(component_to_endpoint)
-    device:set_endpoint_to_component_fn(endpoint_to_component)
     if device:get_field(IS_PARENT_CHILD_DEVICE) == true then
       device:set_find_child(find_child)
     end
+    device:set_component_to_endpoint_fn(component_to_endpoint)
```

Comment:
nit: can these be put in their original location above the `if device:get_field(IS_PARENT_CHILD_DEVICE) == true then` above to reduce the unneccessary diff?

Reply:
Put back in original position in 340df0f5c6c6d95d4d9b7c19e2b70b4b2fb646e2